datasets/compguesswhat/dataset_infos.json

```diff
@@ -0,0 +1,580 @@
+          "_type": "Value"
+        }
+      },
+      "qas": {
+        "feature": {
+          "question": {
+            "dtype": "string",
+            "id": null,
+            "_type": "Value"
+          },
+          "answer": {
+            "dtype": "string",
+            "id": null,
+            "_type": "Value"
+          },
+          "id": {
+            "dtype": "int32",
+            "id": null,
+            "_type": "Value"
+          }
+        },
+        "length": -1,
+        "id": null,
+        "_type": "Sequence"
+      },
+      "objects": {
+        "feature": {
+          "id": {
+            "dtype": "int32",
+            "id": null,
+            "_type": "Value"
+          },
+          "bbox": {
+            "feature": {
+              "dtype": "float32",
+              "id": null,
+              "_type": "Value"
+            },
+            "length": 4,
+            "id": null,
+            "_type": "Sequence"
+          },
+          "category": {
+            "dtype": "string",
+            "id": null,
+            "_type": "Value"
+          },
+          "area": {
+            "dtype": "float32",
+            "id": null,
+            "_type": "Value"
+          },
+          "category_id": {
+            "dtype": "int32",
+            "id": null,
+            "_type": "Value"
+          },
+          "segment": {
+            "feature": {
+              "feature": {
+                "dtype": "float32",
+                "id": null,
+                "_type": "Value"
+              },
+              "length": -1,
+              "id": null,
+              "_type": "Sequence"
+            },
+            "length": -1,
+            "id": null,
+            "_type": "Sequence"
+          }
+        },
+        "length": -1,
+        "id": null,
+        "_type": "Sequence"
+      }
+    },
+    "supervised_keys": null,
+    "builder_name": "compguesswhat",
+    "config_name": "default",
+    "version": {
+      "version_str": "0.1.0",
+      "description": null,
+      "nlp_version_to_prepare": null,
+      "major": 0,
+      "minor": 1,
+      "patch": 0
+    },
+    "splits": {
+      "train": {
+        "name": "train",
+        "num_bytes": 125184123,
+        "num_examples": 46277,
+        "dataset_name": "compguesswhat"
+      },
+      "test": {
+        "name": "test",
+        "num_bytes": 25729944,
+        "num_examples": 9619,
+        "dataset_name": "compguesswhat"
+      },
+      "validation": {
+        "name": "validation",
+        "num_bytes": 25732027,
+        "num_examples": 9716,
+        "dataset_name": "compguesswhat"
+      }
+    },
+    "download_checksums": {
+      "https://www.dropbox.com/s/gawk420r77huc01/compguesswhat_games.zip?dl=1": {
+        "num_bytes": 106007842,
+        "checksum": "8ee244654ff6c9968d0bcccc903e5fd81b90ad7a6dae0f47f0f99861f84a088f"
+      }
+    },
+    "download_size": 106007842,
+    "dataset_size": 176646094,
+    "size_in_bytes": 282653936
+  },
```

Comment:
If I'm not mistaken there are two configs 'compguesswhat-original' and 'compguesswhat-zeroshot', so this one should be removed:
```suggestion
```
It was probably something that was generated before you switched to having two configs with those names

Reply:
Good catch. Yes I just regenerated them. Thanks!